lms/static/js/spec/commerce/receipt_spec.js

```diff
@@ -0,0 +1,182 @@
+define([
+        'jquery', 'jquery.ajax-retry',
+        'js/commerce/views/receipt_view', 'common/js/spec_helpers/ajax_helpers'
+    ],
+    function ($, AjaxRetry, ReceiptView, AjaxHelpers){
+        'use strict';
+        describe("edx.commerce.ReceiptView", function() {
```

Comment:
Nit: don't mix single and double quotes in one file. Our best practice is to use single quotes except when the string itself includes one.